test/packetTest.js

```diff
@@ -215,6 +228,7 @@ function getValue (_type, packet) {
   }
 }
 
+mc.supportedVersions = ['1.19.3']
```

Comment:
seems like your forgot this

you can use -g of mocha to run only some tests

Reply:
Good to know. Have already reverted it